ert/serialization/_serializer.py

```diff
@@ -70,3 +71,26 @@ async def decode_from_path(
         async with aiofiles.open(path, mode="rt", encoding="utf-8") as filehandle:
             contents = await filehandle.read()
             return self.decode(contents, *args, **kwargs)
+
+
+class _ecl_sum_serializer(Serializer):
+    def encode(self, obj: Any, *args: Any, **kwargs: Any) -> str:
+        raise NotImplementedError("not implemented")
+
+    def decode(self, series: str, *args: Any, **kwargs: Any) -> Any:
+        raise NotImplementedError("not implemented")
+
+    async def encode_to_path(
+        self, obj: Any, path: Union[str, Path], *args: Any, **kwargs: Any
+    ) -> None:
+        raise NotImplementedError("not implemented")
+
+    async def decode_from_path(
+        self, path: Union[str, Path], *args: Any, **kwargs: Any
+    ) -> Any:
+        key = kwargs.get("key", None)
```

Comment:
Was just wondering, can we do wild-cards; ie. retrieving multiple keys. Since it seems to get more keys at once we need to use open the same files multiple times :thinking:

Reply:
Should we not wait with wildcards and multiple keys until we have introduced the RecordTree?

Reply:
Sounds good, just that the serializer has nothing to do with the RecordTree...

Reply:
We need a `RecordTree` to do more ;)